tests/collections/test_custom_templates.py

```diff
@@ -58,3 +58,31 @@ def test_hydrate_custom_template(client: Albert):
         # identity checks
         assert hydrated.id == custom_template.id
         assert hydrated.name == custom_template.name
+
+
+def test_create_custom_template_from_seed(
+    caplog,
+    client: Albert,
+    seed_prefix: str,
+    seeded_custom_templates: list[CustomTemplate],
+):
+    """Test creating a new custom template."""
+    seed = seeded_custom_templates[0]
+
+    new_template = CustomTemplate(
+        name=seed_prefix,
+        category=seed.category,
+        data=(
+            seed.data.model_copy(update={"name": seed_prefix}, deep=True)
+            if getattr(seed, "data", None) is not None
+            else None
+        ),
+    )
```

Comment:
can we avoid seeding custom_templates, since we are not using it anywhere, and just define a model here instead, something like 

```suggestion
    new_template = CustomTemplate(
        name=seed_prefix,
        category=TemplateCategory.GENERAL,
        data=GeneralData(name=f"{seed_prefix}-general")
    )
```